kroxylicious-app/src/assembly/kroxylicious-start.sh

```diff
@@ -20,6 +20,10 @@ classpath() {
   echo "${class_path}"
 }
 
+if [ "${KROXYLICIOUS_LOGGING_OPTIONS+set}" != set ]; then
```

Comment:
Bit of a [fiddle](https://pubs.opengroup.org/onlinepubs/9699919799/utilities/V3_chap02.html#tag_18_06_02) to find the right shell expansion to test purely for unset.

| expansion         | parameterSet and Not Null | parameterSet But Null | parameterUnset  |
|-------------------|---------------------------|-----------------------|-----------------|
| ${parameter+word} | substitute word           | substitute word       | substitute null |

At first I tried using `${KROXYLICIOUS_LOGGING_OPTIONS:-default}` but this used the default if there was an empty string supplied like `KROXYLICIOUS_LOGGING_OPTIONS="" kroxylicious-start.sh`.

Reply:
wanted to make sure users has some mechanism to set it to empty string